ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/TransactionConversion.scala

```diff
@@ -23,7 +22,8 @@ import scala.collection.breakOut
 trait TransactionConversion {
 
   def acceptedToDomainFlat(
```

Comment:
Please rename from `acceptedToDomainFlat` to something like `ledgerEntryTxToDomainFlat`.